packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -322,7 +324,7 @@ export class RichText extends Component {
 				} );
 				this.lastContent = this.valueToFormat( linkedRecord );
 				this.lastEventCount = undefined;
-				this.props.onChange( this.lastContent );
+				this.onChange( this.lastContent );
```

Comment:
Those won't be triggered anymore since `this.lastContent` does not have a nativeEvent property. I think this is a mistake, we should keep calling `this.props.onchange( this.lastContent );` in those cases